app/src/main/java/com/example/placefinder/dao/IParkDAO.kt

```diff
@@ -5,6 +5,6 @@ import retrofit2.Call
 import retrofit2.http.GET
 
 interface IParkDAO {
-    @GET("JSON/")
+    @GET("/api/v1/parks")
```

Comment:
Added the end of the JSON service URL to the IParkDAO interface. I followed this video created by the Professor to add the URLs: https://www.youtube.com/watch?v=u9fRy2N9GzE